spec/factories/courses.rb

```diff
@@ -26,6 +26,13 @@
     additional_degree_subject_requirements { true }
     degree_subject_requirements { Faker::Lorem.sentence.to_s }
 
+    accept_pending_gcse { true }
+    accept_gcse_equivalency { true }
+    accept_english_gcse_equivalency { true }
+    accept_maths_gcse_equivalency { false }
+    accept_science_gcse_equivalency { false }
```

Comment:
Is there a particular reaon if not can we user `[true, false].sample`

Reply:
My bad, added the factory in last so didn't check to see if it broke anything. Will fix as suggested 👍

Reply:
okay so had a bit of trouble testing the output of  `[true, false].sample` in the stubbed API response. Therefore, I've packaged these attributes as a trait on the courses factory. Is this okay?